sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2131,6 +2263,19 @@ public partial class PredictiveValue
         public System.DateTimeOffset TimeStamp { get { throw null; } }
         public double Value { get { throw null; } }
     }
+    public partial class PrivateLinkScopedResource
+    {
+        internal PrivateLinkScopedResource() { }
+        public string ResourceId { get { throw null; } }
```

Comment:
Is this property in a format of arm resource id? If so, please change the format of this property to `arm-id`

Reply:
Noop. We want to remove **PrivateLinkScopedResource** from the SDK.

Reply:
Done